proposal/README.md

```diff
@@ -85,45 +85,109 @@ This is a lower priority option as most entities in flyte are immutable and do n
 # Details of each resource
 
 ## Projects
-Support
+Projects are top level entity in Flyte. You can fetch multiple projects or one project using the CLI. Think about projects like namespaces.
+
  - create
+```bash
+$ flytectl create projects --name "Human readable Name of project" --id project-id --labels key=value --labels key=value --description "long string"
+Alternatively
+$ flytectl create project -f project.yaml
+```
+
+```yaml
+project.yaml
+name: Human readable project name
+id: project-x
+labels:
+  - k: v
+  - k1: v1
+description: |
+  Long description
+```
  - get
+```bash
+$ flytectl get projects [project-name] [-o yaml | -o json | default -o table]
+```
  - update
+```bash
+$ flytectl update projects --id project-x ...
+# You can only update one project at a time
+```
 
 ## Tasks
-Support
- - create
  - get
+```bash
+$ flytectl get tasks [task-name] [-o yaml | -o json | default -o table] [--filters...] [--sort-by...] [--selectors...]
```

Comment:
add a limit param here (and other list commands)? this is required by admin

Reply:
Is it a good idea to make it configurable